config/vmdb.tmpl.yml

```diff
@@ -344,6 +344,8 @@ workers:
           :poll_method: :escalate
         :ems_metrics_collector_worker_kubernetes:
           :poll_method: :escalate
+          :hawkular_port: 5000
```

Comment:
The `vmdb.tmpl.yml` config file is used indeed to not hard-code parameters. These hawkular parameters can be configured per ManageIQ deployment (atm), which is the same limitation that we have today for other providers (see `amqp_port`). cc @blomquisg @chessbyte